l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -190,14 +190,14 @@ def _create_targets_for_deep_prediction(
     yaws_offset = np.zeros((num_frames, 1), dtype=np.float32)
     availability = np.zeros((num_frames,), dtype=np.float32)
 
-    for i, (frame, agents) in enumerate(zip(frames, agents)):
+    for i, (frame, frame_agents) in enumerate(zip(frames, agents)):
```

Comment:
Minor: how about agent?

Reply:
Spoke too soon, seeing the contents of agents now. I just liked the notation frames -> frame, agents-> agent. frame_agents is a bit different, but well, minor. and would not know how to improve.

Reply:
yeah..naming is not our top skill, let's put it that way xD